File: Python/1074.py
n = int(input())
array = []
while n > 0:
    array.append(int(input()))
    n-=1

for i in range(0, len(array)):
    x = array[i]
    if x >0:
        if x %2 == 0:
            print("EVEN POSITIVE")
        else:
            print("ODD POSITIVE")
    elif x == 0:
        print("NULL")
    else:
        if x %2 == 0:
            print("EVEN NEGATIVE")
        else:
            print("ODD NEGATIVE")

File: PHP/1074.php
<?php

$n = (int) readline();
$array = [];

while ($n > 0){
	$array[] = (int) readline();
	$n -=1;
}

for ($i = 0; $i < count($array); $i++){
	$x = $array[$i];
	if ($x > 0){
		if ($x % 2 == 0)
			echo ("EVEN POSITIVE\n");
		else
			echo ("ODD POSITIVE\n");
	}
	else if ($x == 0)
		echo ("NULL\n");
	else{
		if ($x % 2 == 0)
			echo("EVEN NEGATIVE\n");
		else
			echo("ODD NEGATIVE\n");
	}
}
?>
